backend/src/libs/packages/server-application/libs/helpers/is-white-route/is-white-route.helper.ts

```diff
@@ -0,0 +1,6 @@
+import { WHITE_ROUTES } from '#libs/packages/server-application/libs/constants/constants.js';
+
+const isWhiteRoute = (route: string): boolean =>
```

Comment:
Please, check B20 in quality criteria

Reply:
would it be ok to name it `checkIsWhiteRoute`?

Reply:
Yep, sounds good